xml/bp_troubleshooting_monitors.xml

```diff
@@ -186,6 +186,76 @@
    <literal>mon.a</literal> has rank 0.
   </para>
  </sect1>
+ <sect1 xml:id="mons-restoring-quorum">
+  <title>Restoring the MONs quorum</title>
+
+  <para>
+   In case the &mon;s cannot form a quorum, &cephadm; is not able to manage the
+   cluster until the quorum is restored. In order to restore the MON quorum,
+   remove unhealthy MONs form the monmap by following these steps:
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Stop all MONs. Log in to each MON host via SSH and run the following
+     command there:
+    </para>
+<screen>&prompt.cephuser;cephadm unit --name mon.`hostname` stop</screen>
+   </step>
+   <step>
+    <para>
+     Identify a surviving monitor by logging in to that host via SSH and
+     running:
+    </para>
+<screen>&prompt.cephuser;cephadm enter --name mon.`hostname`</screen>
+   </step>
+   <step>
+    <para>
+     Extract a copy of the monmap to a file, for example
+     <filename>/tmp/monmap</filename>. Note that
+     <replaceable>MON_ID</replaceable> is usually identical to the string that
+     the <command>hostname</command> command returns:
+    </para>
+<screen>ceph-mon -i <replaceable>MON_ID</replaceable> --extract-monmap <replaceable>/tmp/monmap</replaceable></screen>
+   </step>
+   <step>
+    <para>
+     Remove the non-surviving or problematic monitors. For example, if you have
+     three monitors, <literal>mon.a</literal>, <literal>mon.b</literal>, and
+     <literal>mon.c</literal> where only <literal>mon.a</literal> will survive,
```

Comment:
**will** survive or already **is** surviving? I am not sure of the intention here.